src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2357,7 +2357,6 @@ internal void Run(bool inputLoopIsNested)
                 Dbg.Assert(ui != null, "Host.UI should return an instance.");
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-macos`
Native streams behavior with PowerShell.Error stream.preserves error stream as is with Out-String
```powershell
Expected strings to be the same, but they were different.
Expected length: 33
Actual length:   51
Strings differ at index 4.
Expected: 'foo\n\nbar\n\nbazmiddlefoo\n\nbar\n\nbaz\n'
But was:  'foo\n \n\n \nb...'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeStreams.Tests.ps1: line 52
52:             ($out | Out-String).Replace("`r", '') | Should -BeExactly "foo`n`nbar`n`nbazmiddlefoo`n`nbar`n`nbaz`n"
```
Native streams behavior with PowerShell.Error stream.does not get truncated or split when redirected
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Scripting/NativeExecution/NativeStreams.Tests.ps1: line 62
62:             $e.Count | Should -Be 1
```